db/migrate/20171031201847_create_desks.rb

```diff
@@ -2,6 +2,7 @@ class CreateDesks < ActiveRecord::Migration[5.1]
   def change
     create_table :desks do |t|
       t.belongs_to :grouping, index: true
+      t.references :coordinate
```

Comment:
Don't touch these migrations since theyre not going to be run anymore unless you nuke the DB. Make a new migration to add this column to `Grouping` and `Desk`